OAR/curl/request_doi.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import json,cgi,time
+import httplib2, sys, base64, codecs
+
+res=[]
+retCode=0
+errCode=''
+doi='11623'
+res =  "%s/sci-gaia:%s" % (doi,time.time())
+print "Content-type: application/json\n\n"
+print json.dumps(res)
```

Comment:
Actually is not a request, it just generates a unique identifier for the object based on the timestamp. I'm trying to write a README.md in curl folder that try to better explains how to perform the warmup. What do you think?